image-patching/stopgap-script/nuage_overcloud_full_patch.py

```diff
@@ -299,25 +281,20 @@ def main(args):
 
         cmds_run(['echo "Creating Repo File"'])
         if 'RepoName' in argsDict:
-            create_repo_file(argsDict['RepoName'][0], argsDict['RepoBaseUrl'][0], argsDict['ImageName'][0])
+            create_repo_file(argsDict['RepoName'][0], argsDict['RepoBaseUrl'][0], argsDict['ImageName'][0],argsDict['RpmPublicKey'][0])
```

Comment:
@ckogant, please add readme instructions on how to pass gpgkey while running image patching script

Reply:
done